src/main/kotlin/org/rust/ide/formatter/RustFormattingModelBuilder.kt

```diff
@@ -16,6 +16,7 @@ class RustFormattingModelBuilder : FormattingModelBuilder {
     override fun createModel(element: PsiElement, settings: CodeStyleSettings): FormattingModel {
         val ctx = RustFmtBlockContext.create(settings)
         val block = AbstractRustFmtBlock.createBlock(element.node, null, Indent.getNoneIndent(), null, ctx)
+        /** / com.intellij.formatting.FormattingModelDumper.dumpFormattingModel(block, 2, System.err) // */
```

Comment:
You can also write a local helper function: https://github.com/intellij-rust/intellij-rust/blob/master/src/main/kotlin/org/rust/lang/core/resolve/RustResolveEngine.kt#L456-L465